code/datums/brain_damage/mild.dm

```diff
@@ -0,0 +1,250 @@
+	scan_desc = "phobia"
+	gain_text = ""
+	lose_text = ""
+	var/phobia_type
+	var/next_check = 0
+	var/next_scare = 0
+	var/list/trigger_words
+	//instead of cycling every atom, only cycle the relevant types
+	var/list/trigger_mobs = list()
+	var/list/trigger_objs = list() //also checked in mob equipment
+	var/list/trigger_turfs = list()
+	var/list/trigger_species = list()
+
+/datum/brain_trauma/mild/phobia/New(mob/living/carbon/C, _permanent, specific_type)
+	phobia_type = specific_type
+	if(!phobia_type)
+		phobia_type = pick("spiders", "space", "security", "clowns", "greytide", "lizards")
+
+	gain_text = "<span class='warning'>You start finding [phobia_type] very unnerving...</span>"
+	lose_text = "<span class='notice'>You no longer feel afraid of [phobia_type].</span>"
+	scan_desc += " of [phobia_type]"
+	switch(phobia_type)
+		if("spiders")
+			trigger_words = list("spider","web","arachnid")
+			trigger_mobs = list(/mob/living/simple_animal/hostile/poison/giant_spider)
+			trigger_objs = list(/obj/structure/spider) //includes webs and spiderlings
+		if("space")
+			trigger_words = list("space", "star", "universe", "void")
+			trigger_turfs = list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)
+		if("security")
+			trigger_words = list(" sec ", "security", "shitcurity", "stunbaton", "taser", "beepsky")
+			trigger_mobs = list(/mob/living/simple_animal/bot/secbot)
+			trigger_objs = list(/obj/item/clothing/under/rank/security, /obj/item/clothing/under/rank/warden, /obj/item/clothing/under/rank/head_of_security,\
+				/obj/item/clothing/under/rank/det, /obj/item/melee/baton, /obj/item/gun/energy/taser, /obj/item/restraints/handcuffs, /obj/machinery/door/airlock/security)
+		if("clowns")
+			trigger_words = list("clown", "honk", "banana", "slip")
+			trigger_objs = list(/obj/item/clothing/under/rank/clown, /obj/item/clothing/shoes/clown_shoes, /obj/item/clothing/mask/gas/clown_hat,\
+				/obj/item/device/instrument/bikehorn, /obj/item/device/pda/clown, /obj/item/grown/bananapeel)
+		if("greytide")
+			trigger_words = list("assistant", "grey", "gasmask", "gas mask", "stunprod", "spear", "revolution", "viva")
+			trigger_objs = list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/cattleprod, /obj/item/twohanded/spear,\
+				/obj/item/clothing/mask/gas)
+		if("lizards")
+			trigger_words = list("lizard", "ligger", "hiss", " wag ")
+			trigger_objs = list(/obj/item/toy/plush/lizardplushie, /obj/item/reagent_containers/food/snacks/kebab/tail, /obj/item/severedtail,\
+				/obj/item/reagent_containers/food/drinks/bottle/lizardwine)
+			trigger_mobs = list(/mob/living/simple_animal/hostile/lizard) //they're hostile! of course they're scary!
+			trigger_species = list(/datum/species/lizard)
+		if("spooky skeletons")
+			trigger_words = list("skeleton", "rattle me bones", "milk", "xylophone", "bone", "calcium", "i want to get off mr bones wild ride", "the ride never ends")
```

Comment:
Should just shorten it to "mr bones wild ride".

Reply:
What @Slignerd said